src/store/storage-controller.ts

```diff
@@ -111,19 +109,43 @@ export class StorageController {
     return messages;
   }
 
-  public static async delete(
-    messageStore: MessageStore,
-    dataStore: DataStore,
+  public async delete(
```

Comment:
maybe `deleteMessage` to differentiate from `deleteEvents`?

Reply:
Addressed in https://github.com/TBD54566975/dwn-sdk-js/pull/338/commits/42f36fb0385704079ff7f0a11a32f953d80a25c7